server/index.js

```diff
@@ -106,26 +73,6 @@ app.delete('/houses/:houseId', (req, res) => {
   // pass in house id
 });
 
-// create
-app.post('/agents/:agentId', (req, res) => {
-  // need id (unless autogenerated), company, name, phone, recent sales, reviews, image, premiere
-});
-
-// read
-app.get('/agents/:agentId', (req, res) => {
-  // pass in agent id
-});
-
-// update
-app.patch('/agents/:agentId', (req, res) => {
-  // pass in agent id and one or more of: company, name, phone, recent sales, reviews, image, premiere
-});
-
-// delete
-app.delete('/agents/:agentId', (req, res) => {
-  // pass in agent id
-});
-
 app.listen(PORT, () => {
   console.log(`Hidey-Ho Cap'n, we are now serving on port ${PORT}!`);
 });
```

Comment:
I would recommend moving the routes  into a controller.js file to adhere to the MVC Framework

Reply:
@PlasmaIntec I refactored if you'd like to take a look

Reply:
@elizabethcan changes look good- although is your database performing well even without pooling connections?